### PR TITLE
Fix/GitHub pagination

### PR DESCRIPTION
**Pull request by saihtaungkham**
Improve the Github Pagination Logic.
Cherry Pick one fix from develop branch.

### CODE REVIEW

File: src/main/python/mlsearch/api_requester.py [hunk 1]
```diff
@@ -11,11 +11,13 @@
 import html
 import random
 import collections
+import math
 
 # import scholarly
 
 ErrorType = collections.namedtuple("ErrorType", "reason status")
 
+
```
Comment (rhezaharliman) [inline src/main/python/mlsearch/api_requester.py:20]: Unnecessary empty line? 🤔

**Review by rhezaharliman — APPROVED** [review]
Thanks, Sai! I haven't checked the behavior, but since the calculation also looks good, I think it'll be fine.
It'll be great though if the variable is easier to understand (or adds some documentation). 
For example, `item_per_page` will be easier to understand rather than `per_page`. or `firs_page_item_index`,  etc. It's just a matter of style.

File: src/main/python/mlsearch/api_requester.py (hunk 1 above)
Comment (saihtaungkham) [reply src/main/python/mlsearch/api_requester.py:20]: This is the `black` formatter style. Two blank lines before the class name.

Comment (saihtaungkham) [reply src/main/python/mlsearch/api_requester.py:20]: `per_page` is derived from Github API. I will modify it. Thanks.

**Comment by saihtaungkham** [comment]
I fixed the variable name and some code formatting is applied. Thanks for checking my formulae too.

**Review by rhezaharliman — APPROVED** [review]
Thanks, Sai! Looks good for me :)

**Pull request merged**